docs/environment-integrations/mujoco.md

```diff
@@ -0,0 +1,90 @@
+# MuJoCo Integrations
+
+### Installation
+
+Install Sample-Factory with MuJoCo dependencies with PyPI:
+
+```
+pip install sample-factory[mujoco]
+```
+
+### Running Experiments
+
+Run MuJoCo experiments with the scripts in `sf_examples.mujoco_examples`. The default parameters have been chosen to match CleanRL's results in the report below.
+
+To train a model in the `Ant-v4` enviornment:
+
+```
+python -m sf_examples.mujoco_examples.train_mujoco --algo=APPO --env=mujoco_ant --experiment="Experiment Name"
```

Comment:
I'd say <experiment_name> instead of "Experiment name" (a more standard way to show that this is a placeholder)